nsxt/utils_test.go

```diff
@@ -112,6 +112,10 @@ func getTestCertificateName(isClient bool) string {
 	return os.Getenv("NSXT_TEST_CERTIFICATE_NAME")
 }
 
+func getTestLicenseKey() string {
+	return os.Getenv("NSXT_LICENSE_KEY")
```

Comment:
lets call it NSXT_TEST_LICENSE_KEY to align with other test env settings. Thanks!